test/e2e/kubectl.go

```diff
@@ -1044,6 +1045,8 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		jobName := "e2e-test-rm-busybox-job"
 
 		It("should create a job from an image, then delete the job [Conformance]", func() {
+			// The rkt runtime doesn't support attach, see #23335
+			framework.SkipIfContainerRuntimeIs("rkt")
```

Comment:
I'm not sure; is attach required for conformance? The test that only tests 'attach' is not tagged as conformance, but this one incidentally tests attach, and that's the only part of it which rkt can't do.

I'd prefer if `attach` were not part of Conformance (related issue: #23335) and this test were either not conformance or were reworked to not use attach, but if attach is required for conformance, rkt isn't and hasn't conformed.